pages/pipelines/conditionals.md.erb

```diff
@@ -59,7 +59,7 @@ The below expressions are supported by the `if` attribute:
  			<td><code>'feature-branch' "feature-branch"</code></td>
  		</tr>
 		<tr>
- 			<td>Booleans</td>
+			<td>Boolean</td>
```

Comment:
"Boolean literals"? Or even "Literals" and fold in `null` as well? Seems a bit odd that that one's singular too.